src/main/java/com/savvato/tribeapp/controllers/AttributesAPIController.java

```diff
@@ -38,6 +39,16 @@ public class AttributesAPIController {
     AttributesAPIController() {
     }
 
+    @GetNumberOfUsersWithAttribute
+    @GetMapping("/total/{attributeId}")
+    public ResponseEntity<Integer> getNumberOfUsersWithAttribute(
```

Comment:
Use the GenericMessageDTO here, and return a JSON object with this Integer value.